packages/trader/src/Modules/CFD/Components/cfd-demo-account-display.tsx

```diff
@@ -1,9 +1,46 @@
 import React from 'react';
 import { localize } from '@deriv/translations';
 import specifications from 'Modules/CFD/Constants/cfd-specifications';
-import { CFDAccountCard } from './cfd-account-card.jsx';
+import { CFDAccountCard } from './cfd-account-card';
```

Comment:
No need to do this in this PR!

Reply:
@matin-binary, but how to be with issues of versel and ci?